docs/buildah-bud.md

```diff
@@ -657,6 +660,25 @@ will convert /foo into a `shared` mount point.  The propagation properties of th
 mount can be changed directly. For instance if `/` is the source mount for
 `/foo`, then use `mount --make-shared /` to convert `/` into a `shared` mount.
 
+## BUILD TIME VARIABLES
+
+The ENV instruction in a Containerfile can be used to define variable values.  When the image
+is built, the values will persist in the container image.  At times it is more convenient to 
+change the values in the Containerfile via a command-line option rather than changing the 
+values within the Containerfile itself.
+
+The following variables can be used in conjunction with the `--build-arg` option to override the
+values set in the Containerfile.
+
+  * HTTP_PROXY
+  * HTTPS_PROXY
+  * FTP_PROXY
+  * NO_PROXY
```

Comment:
What about FROM?

Reply:
next paragraph, slightly different handling as these are all specified like `ENV HTTP_PROXY="http://127.0.0.1:8321", but let me tease that out a bit more to be clear.